provisioning_virtual_machines_and_hosts/_topics/catalogs_and_services.md

```diff
@@ -372,6 +372,125 @@ Service methods have been split based on purpose.
 
 7.  Click **Save** or **Add** if you are adding this to a new instance.
 
+### Provisioning Dialog
+
+The following example shows how to create a Provisioning Dialog that modifies the parameters for provisioning a small, medium, or large service.
+
+The `provision-vm-small-med-large.asl` workflow runs for every provisioned VM and is responsible for modifying the VM size. This workflow is the `<ManageIQ>` service Provisioning Entry Point.
+
+You need to set the ManageIQ Provisioning service entry point to workflow script. For example, this workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
```

Comment:
We probably should not use the word ManageIQ here as it should be productized.  You can productize with `{{ site.data.product.title }}` or, more simply, just don't mention the product name at all because it's redundant.